internal/pkg/service/buffer/store/model/slice.go

```diff
@@ -13,6 +13,7 @@ import (
 type Slice struct {
 	key.SliceKey
 	State       slicestate.State `json:"state" validate:"required,oneof=opened closing closed uploading uploaded failed"`
+	IsEmpty     bool             `json:"isEmpty,omitempty"`
```

Comment:
Ak je slice empty, tak jej nenastavujeme statistiky (lebo tam ocakavame, ze count >0).
Namiesto toho nastavime flag `IsEmpty`, ktory potom vyuzije `uploader` a slice sa preskoci.

Reply:
A znova sa to vyuzije pri generovani manifestu.

Reply:
Prakticky, iba posledna slice vo file moze byt prazdna, ... ak sa zatvara file a v tom case este nic neprislo do poslednej slice.